### PR TITLE
Simplify the Port interface.

### PR DESCRIPTION
**Pull request by adleong**
I kept getting confused by the VarUp and ActUp type aliases.  This change allows us to get rid of them and hopefully improves readability.

### CODE REVIEW

File: k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala [hunk 1]
```diff
@@ -113,11 +113,9 @@ class EndpointsNamer(idPrefix: Path, mkApi: String => NsApi) extends Namer {
 private object EndpointsNamer {
   val PrefixLen = 3
 
-  type VarUp[T] = Var[T] with Updatable[T]
-  type ActUp[T] = VarUp[Activity.State[T]]
```
Comment (klingerf) [inline k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala]: These type aliases also exist in CatalogNamer.scala -- do you think it's worth trying to remove them from there as well?

Comment (adleong) [inline k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala]: Probably.  I plan to take a pass through the consul namer as well.

**Comment by klingerf** [comment]
⭐ 🐒

**Comment by olix0r** [comment]
:star:

**Pull request merged**